docs/source/usage.rst

```diff
@@ -1,47 +1,75 @@
 Usage
 =====
 
-Automatic Notebook Versioning
------------------------------
+Data scientists and notebook users may develop locally on their system or save their notebooks to off-site or cloud
+storage. Additionally, they will often create a notebook and then over time make changes and update it. As
+they work, it's helpful to be able to **store versions** of a notebook. When making changes to the content and
+calculations over time, a data scientist using Bookstore can now request different versions from the remote storage,
+such as S3, and **clone** the notebook to their local system.
 
-Every *save* of a notebook creates an *immutable copy* of the notebook on object storage.
+                                             *store*
 
-To simplify implementation, we currently rely on S3 as the object store, using
+Data scientist saves to Local System ------------------> Remote Data Store (i.e. S3) Workspace
+
+                                             *clone*
+
+User requests a notebook to use locally <-------------- Remote Data Store (i.e. S3) Workspace
+
+
+After some time working with a notebook, the data scientist may want to save or share a polished notebook version with
+others. By **publishing a notebook**, the data scientist can display and share work that others can use at a later time.
+
+How to store and clone versions
+-------------------------------
+
+Bookstore uses automatic notebook version management and specific storage paths when storing a notebook.
+
+Automatic Notebook Version Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Every *save* of a notebook creates an *immutable copy* of the notebook on object storage. Initially, Bookstore
```

Comment:
While it is an immutable copy, I feel like we might want to be clearer that it is overwriting the file in place. It relies on versioned s3 buckets to handle the versioning.

Reply:
Good point on the overwrite. Will add.

Reply:
Added to the following paragraph.